src/main/kotlin/frc/robot/subsystems/elevator/ElevatorConstants.kt

```diff
@@ -0,0 +1,20 @@
+package frc.robot.subsystems.elevator
+
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.measure.Distance
+import frc.robot.lib.Gains
+import frc.robot.lib.selectGainsBasedOnMode
+import kotlin.math.PI
+
+const val MAX_HEIGHT = 1.3
+const val GEAR_RATIO = (1.0 / 12.0) * (42.0 / 48.0)
+const val FIRST_STAGE_RATIO = 2.0
+val SPROCKET_RADIUS: Distance = Units.Millimeters.of(36.4 / 2.0)
+val ROTATIONS_TO_CENTIMETER = GEAR_RATIO * FIRST_STAGE_RATIO * (SPROCKET_RADIUS.`in`(Units.Centimeter) * 2 * PI)
```

Comment:
Should be a measure.

Reply:
I don't think it should be a function, it's a simple conversion. But I'm resolving this and let's continue discussion on https://github.com/Galaxia5987/robot-2025/pull/1/files#r1917084517.